docs/grid_exchange_formats/cgmes/export.md

```diff
@@ -2,111 +2,106 @@
 
 <span style="color: red">TODO</span>
 
-Please note that PowSyBl always exports CGMES networks as CGMES Node/Breaker networks without considering the topology 
-level of the PowSyBl network.
+Please note that PowSyBl only ever exports CGMES networks as CGMES Node/Breaker networks without consideration of the topology level of the PowSyBl network.
 
 ## Conversion from PowSyBl grid model to CGMES
 
-The following sections describe in detail how each supported PowSyBl network model object is converted to CGMES network 
-components.
+The following sections describe in detail how each supported PowSyBl network model object is converted to CGMES network components.
 
 ### Battery
 
-PowSyBl [`Batteries`](../../grid_model/network_subnetwork.md#battery) are exported as CGMES `SynchronousMachine` with CGMES `HydroGeneratingUnits`.
+PowSyBl [`Batteries`](../model/index.md#battery) are exported as CGMES `SynchronousMachine` with CGMES `HydroGeneratingUnits`.
```

Comment:
Why this link? Same remark in the whole file.

Reply:
Fixed